Tests/Rules/UseConsistentIndentation.tests.ps1

```diff
@@ -6,17 +6,19 @@ Import-Module (Join-Path $testRootDirectory "PSScriptAnalyzerTestHelper.psm1")
 
 $indentationUnit = ' '
 $indentationSize = 4
+$ruleConfiguration = @{
```

Comment:
Heh, looks like you used the formatter to format your formatter test script, nice!